spec/models/allergy_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Allergy, type: :model do
+  before(:all) do
+    @allergy1 = FactoryBot.create(:allergy)
```

Comment:
please, do not name variables like that
use something like `allergy` and `another_allergy`
or `valid_allergy` and `invalid_allergy`

Reply:
Ok, renamed.